releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,60 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 3.0.0-preview - June 17, 2022
+
+### Dynamic Features 
+
+Dynamic features are a tool that can be used to surface different variants of a feature to different segments of an audience. Previously, this library only worked with feature flags. Feature flags are limited to boolean values, as they are either enabled or disabled. Dynamic features have dynamic values. They can be string, int, a complex object, or any other type. 
+
+```cs
+//
+// Modify view based off multiple possible variants
+model.BackgroundUrl = dynamicFeatureManager.GetVariantAsync<string>("HomeBackground", cancellationToken);
+
+return View(model);
+```
+
+For more details read [here](https://github.com/microsoft/FeatureManagement-Dotnet/blob/main/README.md#dynamic-features). 
```

Comment:
I'm wondering if it makes sense to link the release/v3 branch readme? We might change main readme in future, but these notes will continue to point to release/v3 version.

Reply:
I think that's a good idea. I'll update it.